### ISSUE TITLE
uglify-js version in package.json is incompatible with NPM

### ISSUE THREAD

**Issue by howardroark** [macek/jquery-serialize-object#52]
I'm using NPM version 2.1.18
```
npm ERR! Darwin 14.0.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.10.31
npm ERR! npm  v2.1.18
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: uglify-js@'>=2.4.32 <3.0.0'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["0.0.1","0.0.2","0.0.3","0.0.4","0.0.5","1.0.1","1.0.2","1.0.3","1.0.4","1.0.5","1.0.6","1.0.7","1.1.0","1.1.1","1.2.0","1.2.1","1.2.2","1.2.3","1.2.4","1.2.5","1.2.6","1.3.0","1.3.1","1.3.2","1.3.3","1.3.4","2.2.0","2.2.1","2.2.2","2.2.3","2.2.4","2.2.5","2.3.0","2.3.1","2.3.2","2.3.3","2.3.4","2.3.5","2.3.6","1.3.5","2.4.0","2.4.1","2.4.2","2.4.3","2.4.4","2.4.5","2.4.6","2.4.7","2.4.8","2.4.9","2.4.10","2.4.11","2.4.12","2.4.13","2.4.14","2.4.15","2.4.16"]
npm ERR! notarget 
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! Please include the following file with any support request:
```

Seems that the latest version is `2.4.16`

**Comment by macek** [comment]
Are you trying to rebuild the plugin?

Or are you having trouble with uglifying jquery.serialize-object.js within a bundle of your own?

**Comment by howardroark** [comment]
I was building it.  You can see what I had to do to fix it in #53.

**Comment by macek** [comment]
@howardroark sorry about the delay here.

Version [2.4.4](https://github.com/macek/jquery-serialize-object/releases/tag/2.4.4) fixed the `uglify-js` dependency
